cypress/support/commands.js

```diff
@@ -1,3 +1,7 @@
+/// <reference types='cypress' />
+
+import { faker } from '@faker-js/faker';
```

Comment:
The import statement from the 'faker' package is incorrect. It should be 'import faker from 'faker'', not 'import { faker } from '@faker-js/faker''